admin/class-boldgrid-backup-admin-auto-updates.php

```diff
@@ -0,0 +1,244 @@
+<?php
+/**
+ * File: class-boldgrid-backup-admin-auto-updates.php
+ *
+ * @link       https://www.boldgrid.com
+ * @since      SINCEVERSION
+ *
+ * @package    Boldgrid_Backup
+ * @subpackage Boldgrid_Backup/admin
+ * @copyright  BoldGrid
+ * @version    SINCEVERSION
+ * @author     BoldGrid <support@boldgrid.com>
+ */
+
+use Boldgrid\Library\Library\Plugin\Plugins;
+
+/**
+ * Class: Boldgrid_Backup_Admin_Auto_Updates.
+ *
+ * @since 1.2
+ */
+class Boldgrid_Backup_Admin_Auto_Updates {
+	/**
+	 * Settings.
+	 *
+	 * @since SINCEVERSION
+	 * @var array
+	 */
+	public $settings;
+
+	/**
+	 * Active Plugins.
+	 *
+	 * @since SINCEVERSION
+	 * @var array
+	 */
+	public $plugins = array();
+
+	/**
+	 * Themes.
+	 *
+	 * @since SINCEVERSION
+	 * @var array
+	 */
+	public $themes = array();
+
+	/**
+	 * Active Plugins.
+	 *
+	 * @since SINCEVERSION
+	 * @var Boldgrid_Backup_Admin_Core
+	 */
+	public $core;
+
+	/**
+	 * Constructor.
+	 *
+	 * @since SINCEVERSION
+	 */
+	public function __construct() {
+
+		$this->set_settings();
+		$plugins       = new \Boldgrid\Library\Library\Plugin\Plugins();
+		$this->plugins = $plugins->getAllPlugins();
+		$this->themes  = new \Boldgrid\Library\Library\Theme\Themes();
+
+		add_filter( 'automatic_updater_disabled', '__return_false' );
```

Comment:
I had initially tried adding them there, however it didn't trigger them the way it needed to be done. I cannot remember why it wasn't working when placed there, which is why it was done here within the class.

Reply:
> however, this one in particular:
add_filter( 'automatic_updater_disabled', '__return_false' );
won't work using the $this->loader->add_action() function, since ( as far as I can tell ), doesn't allow me to set the filter to '_return_false'

If that's the only one that won't work with `$this->loader->add_action()`, that's fine, just add the `add_filter( 'automatic_updater_disabled', '__return_false' );` call instead. But I'd like to group them in that other file because that's the standard we have for adding filters.